### ISSUE TITLE
Enemy Missionary can convert my city with single spread religion

### ISSUE THREAD

**Issue by PerfectDark99** [LoneGazebo/Community-Patch-DLL#8421]
_1. Mod version (X.Y.Z, e.g. 1.2.1):_
1.2.1

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
An Aztec missionary can convert my city with a single Spread religion action even though I used a Inquisitor couple turns before and had one stationed in the city. (Note: I used the inquisitor because that conversion from single missionary happened 2 times before already.

Maybe I'm missing some intended feature here but it really looks like a weird bug.

_4. Steps to reproduce (optional):_

Load the attached save game, complete the turn and then watch the Aztec missionary convert Surubaya.

--------------------------
[AutoSave_0270 AD-1800.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/7612039/AutoSave_0270.AD-1800.zip)
-
![Civ5Screen0084](https://user-images.githubusercontent.com/58620501/143691466-f1a31829-a376-442b-a06a-35b9758a9f64.jpg)

**Comment by RecursiveVision** [comment]
@ilteroi So people seem fairly unhappy with the new religious pressure system. Common complaints are:
- Inquisitors are useless
- Passive pressure is too weak for the player and too strong for the AI
- Mechanics are opaque and poorly understood

**Comment by ilteroi** [comment]
what is new about it? i did not invent it either. do whatever you want

**Comment by RecursiveVision** [comment]
Oh, I thought you modified it a while back. Never mind then, sorry

**Comment by psparky** [comment]
I don't think there is a bug here. If multiple inquisitors have been used, and possibly an enemy Great Prophet at some time, then the accumulated pressure might be very low. Under those circumstances, the pressure from a missionary could be enough to overpower the current majority religion's accumulated pressure. Can you flip it back with your own missionary?

As for opaque mechanics, perhaps we should consider displaying the accumulated pressure, which determines the number of followers, in the UI (maybe as an option). I did this some time ago in my old version when bug hunting, and I liked it enough to keep it.

![Civ5Screen0003](https://user-images.githubusercontent.com/34212653/143772087-41f3bda9-2c89-4314-aa7d-6cdd52930bae.jpg)

**Comment by PerfectDark99** [comment]
Thanks for your reply psparky. This explain a lot and I indeed was not aware of this hidden mechanic. The accumulated pressure indeed would be helpful.

Given your input I simply spawned a couple missionaries and converted my own city (add pressure) and indeed afterwards the AI was not able to put much of a dent into followers of my religion.

I then also agree with the points of  RecursiveVision. Inquisitors hence seem underpowered. Applying them should not remove all previous pressure.
